src/solvers/dg/dg.jl

```diff
@@ -289,6 +289,14 @@ end
 
 DGSEM(polydeg, surface_flux=flux_central, volume_integral::AbstractVolumeIntegral=VolumeIntegralWeakForm()) = DGSEM(Float64, polydeg, surface_flux, volume_integral)
 
+function DGSEM(; RealT::DataType=Float64,
```

Comment:
```suggestion
# Convenience constructor that allows to pass all parameters in any order as keyword arguments
function DGSEM(; RealT::DataType=Float64,
```

Reply:
I added another comment in 44abf1d9. In general, I don't think comments should explain *what* is done but focus on *why* it is done this way. The *what* is something everybody can figure out more or less easily. The *why* is not necessarily clear and should be explained in a comment such that people revisiting the code know whether or not they should touch specific parts. In addition, understanding *why* often helps for understanding *what*.

Reply:
Yes, absolutely. I like your comment much better than mine.